addons/portal/static/src/js/portal.js

```diff
@@ -283,6 +283,93 @@ publicWidget.registry.portalSecurity = publicWidget.Widget.extend({
 
 });
 
+/**
+ * Defining what happens when you click the "Log Out From All Devices" 
+ * on the "/my/security" page.
+ */
+publicWidget.registry.RevokeSessionsButton = publicWidget.Widget.extend({
+    selector: '#portal_revoke_all_sessions_popup',
+    events: {
+        click: '_onClick',
+    },
+
+    async _onClick(e) {
+        const rpc =this.proxy('_rpc')
+        const wrapped =this._rpc({
+            model: 'res.users',
+            method: 'api_key_wizard',
+            args: [session.user_id],
+        });
+        await handleRevokeSessions(rpc, wrapped, e);
+        window.location.href = "/web/session/logout?redirect=/";
+}
+});
+
+function handleRevokeSessions(rpc, wrapped,e) {
+        return wrapped.then((r) => {
+            if (!_.isMatch(r, {type: 'ir.actions.act_window', res_model: 'res.users.identitycheck'})) {
+                return r;
+            }
+        const check_id = r.res_id
+        var $content = $(qweb.render("portal.revoke_all_devices_popup_template"))
+
+        e.preventDefault();
+        return new Promise((resolve, reject) => {
+            var dialog = new Dialog(this, {
+                title: _t("Log out from all devices ?"),
+                $content,
+                buttons: [{
+                    text: _t("Log Out From All Devices"), classes: 'btn btn-primary',
```

Comment:
```suggestion
                    text: _t("Log out from all devices"), classes: 'btn btn-primary',
```

People Do Not Actually Write Like This :)
(though here it matters less because right now, buttons are all-caps but that won't be the case in the new theme so please change this indeed)